release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3023,6 +3023,60 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.22 --pullspecs
+----
+
+[id="ocp-4-18-22-enhancements_{context}"]
+==== Enhancements
+
+* The readiness probes for the API server (`/readyz` endpoint) have been modified to exclude `etcd` checks. This modification prevents client connections from being closed if `etcd` is temporarily unavailable. As a result, `etcd` will be ready again before a client connection times out, enabling the client connections to persist through a brief `etcd` unavailability. This persistence minimizes temporary API server outages. (link:https://issues.redhat.com/browse/OCPBUGS-49749[OCPBUGS-49749])
+
+[id="ocp-4-18-22-known-issues_{context}"]
+==== Known issues
+
+* Stale Source Network Address Translations (SNATs) or routing policies might occur in the following circumstances:
+** You are upgrading from 4.17 to 4.18 during an update of the OVN-Kubernetes image.
+** During the upgrade, if a pod on another system that was selected by an egress IP was deleted when the `ovnkube-node` pod was not running. (link:https://issues.redhat.com/browse/OCPBUGS-59531[OCPBUGS-59531])
+
+[id="ocp-4-18-22-bug-fixes_{context}"]
+==== Bug fixes
+
+* Before this update, destroying a cluster in the unsupported region `mx-central-1` caused the destroyer to fail to find a partition and not exit. As a consequence, you could not destroy an {product-title} cluster in the `mx-central-1`region due to constant error reporting. With this release, the destroyer does not report errors for the unsupported region `mx-central-1`, which enables the successful destruction of a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-56177[OCPBUGS-56177])
+
+* Before this update, combined specification and status updates lists triggered unnecessary firmware upgrades, which caused system downtime. With this release, a firmware upgrade optimization skips unnecessary firmware upgrades. (link:https://issues.redhat.com/browse/OCPBUGS-56766[OCPBUGS-56766])
+
+* Before this update, the `console-telemetry` plugin received a `Forbidden` error due to using the wrong API endpoint for tracking usage. As a consequence, the `Forbidden error in console-telemetry-plugin usage tracking` error occurred. With this release, the `console-telemetry` plugin posts usage data to `/api/metrics/usage` instead of `/metrics/usage`. As a result, the `console-telemetry`plugin does not receive a `Forbidden` error, which ensures accurate usage tracking. (link:https://issues.redhat.com/browse/OCPBUGS-58364[OCPBUGS-58364])
```

Comment:
This seems off:
> the `Forbidden error in console-telemetry-plugin usage tracking` error

Is that a paste or formatting issue? Or is the error really called "Forbidden error in console-telemetry-plugin usage tracking"?

Reply:
also missing space in here is causing rendering issues
> the `console-telemetry`plugin does not